lms/djangoapps/notes/api.py

```diff
@@ -148,7 +148,7 @@ def create(request, course_key):
     return ApiResponse(http_response=response, data=None)
 
 
-def read(request, course_key, note_id):  # pylint: disable=unused-argument (course_key)
+def read(request, course_key, note_id):  # pylint: disable=unused-argument
```

Comment:
I like that these explain what is being disabled. Maybe we can move the comments to the line above explaining what is the unused_argument?